src/workers/data-importer.ts

```diff
@@ -154,7 +154,7 @@ export class DataImporter {
     return this.queue.length() >= this.maxQueueSize;
   }
 
-  isUnbundleableItem(item: ImportableItem): item is UnbundleableItem {
+  txHasIndexProperty(item: ImportableItem): item is TxWithParentIndex {
```

Comment:
This should just be `hasIndexPropery`. It's not necessarily a transaction.

Reply:
changed